syntax = "proto3";
package form_schema_version;




import "google/protobuf/empty.proto";

/*
DeleteFormSchemaVersion请求
*/
message DeleteFormSchemaVersionRequest {
    /*
    表单实例ID
    */
    string formId = 1 ;
    /*
    表单版本实例ID(多个版本用';'分隔)
    */
    string versionIds = 2 ;
}

/*
DeleteFormSchemaVersionApi返回
*/
message DeleteFormSchemaVersionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

